AnimView/frmMain.cpp

```diff
@@ -95,232 +111,226 @@ frmMain::frmMain()
+  pSidebarSizer->Add(pLayer12, fillSizerFlags);
 
   wxStaticBoxSizer* pMoodOverlay =
       new wxStaticBoxSizer(wxVERTICAL, this, L"Overlays");
-  pMoodOverlay->Add(new wxCheckBox(this, ID_DRAW_MOOD, L"Draw mood overlay"), 0,
-                    wxEXPAND | wxALL, 1);
+  pMoodOverlay->Add(new wxCheckBox(this, ID_DRAW_MOOD, L"Draw mood overlay"),
+                    fillBorderSizerFlags);
   wxBoxSizer* pMoodRow = new wxBoxSizer(wxHORIZONTAL);
   pMoodRow->Add(
       new wxStaticText(this, wxID_ANY, L"Marker position (click to move it):"),
-      0, wxEXPAND | wxRIGHT, 2);
+      wxSizerFlags(0).Expand().Border(wxRIGHT, 2));
   pMoodRow->Add(
-      m_txtMoodPosition[0] = new wxTextCtrl(this, wxID_ANY, L"{0, 0}"), 1,
-      wxEXPAND | wxRIGHT, 1);
+      m_txtMoodPosition[0] = new wxTextCtrl(this, wxID_ANY, L"{0, 0}"),
+      wxSizerFlags(1).Expand().Border(wxRIGHT, 1));
   pMoodRow->Add(
       m_txtMoodPosition[1] = new wxTextCtrl(this, wxID_ANY, L"{0, 0, \"px\"}"),
-      1, wxEXPAND);
-  pMoodOverlay->Add(pMoodRow, 1, wxEXPAND | wxALL, 2);
+      wxSizerFlags(1).Expand());
+  pMoodOverlay->Add(pMoodRow, wxSizerFlags(1).Expand().Border(wxALL, 2));
   pMoodOverlay->Add(
-      new wxCheckBox(this, ID_DRAW_COORDINATES, L"Draw tile coordinates"), 0,
-      wxEXPAND | wxALL, 0);
-  pSidebarSizer->Add(pMoodOverlay, 0, wxEXPAND | wxALL, 0);
+      new wxCheckBox(this, ID_DRAW_COORDINATES, L"Draw tile coordinates"),
+      fillSizerFlags);
+  pSidebarSizer->Add(pMoodOverlay, fillSizerFlags);
   m_bDrawMood = false;
   m_bDrawCoordinates = false;
   m_iMoodDrawX = 0;
   m_iMoodDrawY = 0;
 
   for (int iLayer = 0; iLayer < 13; ++iLayer) {
-    wxCheckBox* pCheck = wxDynamicCast(FindWindow(ID(iLayer, 0)), wxCheckBox);
+    wxCheckBox* pCheck =
+        wxDynamicCast(FindWindow(make_id(iLayer, 0)), wxCheckBox);
     if (pCheck != nullptr) {
       pCheck->SetValue(true);
       m_mskLayers.set(iLayer, 0);
     }
   }
 
-  Connect(ID(0, 0), ID(12, 24), wxEVT_COMMAND_CHECKBOX_CLICKED,
+  Connect(make_id(0, 0), make_id(12, 24), wxEVT_COMMAND_CHECKBOX_CLICKED,
           (wxObjectEventFunction)&frmMain::_onToggleMask);
-#undef ID
 
   wxStaticBoxSizer* pSearch = new wxStaticBoxSizer(wxVERTICAL, this, L"Search");
   wxBoxSizer* pSearchButtons = new wxBoxSizer(wxHORIZONTAL);
-  pSearchButtons->Add(new wxButton(this, ID_SEARCH_LAYER_ID, L"Layer/ID"), 0,
-                      wxALL, 1);
-  pSearchButtons->Add(new wxButton(this, ID_SEARCH_FRAME, L"Frame"), 0, wxALL,
-                      1);
-  pSearchButtons->Add(new wxButton(this, ID_SEARCH_SOUND, L"Sound"), 0, wxALL,
-                      1);
+  pSearchButtons->Add(new wxButton(this, ID_SEARCH_LAYER_ID, L"Layer/ID"),
+                      buttonSizerFlags);
+  pSearchButtons->Add(new wxButton(this, ID_SEARCH_FRAME, L"Frame"),
+                      buttonSizerFlags);
+  pSearchButtons->Add(new wxButton(this, ID_SEARCH_SOUND, L"Sound"),
+                      buttonSizerFlags);
```

Comment:
Sure these buttonSizer things are right, it's missing a `wxALIGN_CENTER_VERTICAL`

Reply:
?

Reply:
Looks good to me. I thought it was better to be consistent with our buttons.

Reply:
Likely a good idea, I just verified whether the flags were the same before and after the change, and these failed that test :)